third_party/txt/src/txt/paragraph.cc

```diff
@@ -423,6 +423,58 @@ bool Paragraph::ComputeBidiRuns(std::vector<BidiRun>* result) {
   return true;
 }
 
+void Paragraph::ComputeStrut(StrutMetrics* strut, SkFont& font) {
+  strut->ascent = 0;  // Positive value to keep signs clear.
```

Comment:
Remove this comment and just keep the one on the `struct StrutMetrics` declaration